### ISSUE TITLE
Does this demo still work now in 2016?

### ISSUE THREAD

**Issue by yangxuan8282** [sandropaganotti-zz/isochrone-with-google-map#1]
I have tried the demo myself, with default settings(Nantes, 10 minutes). 
After waiting about 5 minutes, nothing happen.

**Comment by grhabyt** [comment]
I don't think so. I couldn't get it to work. The Google maps API has changed a great deal in the last seven years.

**Comment by dugwood** [comment]
I've rebooted the project here if you want: https://github.com/dugwood/isochrone-isodistance-with-google-maps
